src/styles/navigation.css

```diff
@@ -0,0 +1,60 @@
+.navBar {
+  display: flex;
+  align-items: center;
+  box-sizing: border-box;
+  padding: 0 40px;
+  height: 50px;
+  min-height: 80px;
+  background-color: rgb(255, 255, 255);
+  width: 100%;
+}
+
+.Navtitle {
+  width: 30%;
+  font-size: 1.875rem;
+  font-family: 'Montserrat', sans-serif;
+  font-weight: 700;
+  color: #0290ff;
+}
+
+.navLinks {
+  padding-right: 160px;
+  margin: 7px;
+  width: 55%;
+  padding-left: 10px;
+  color: #121212;
+}
+
+.link {
+  font-size: 0.813rem;
+  font-weight: 500;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: 1.9px;
+  color: #121212;
+  opacity: 0.5;
+}
+
+.active {
+  font-size: 0.813rem;
+  font-family: 'Montserrat', sans-serif;
+  font-weight: 500;
+  letter-spacing: 1.9px;
+  color: #121212;
+  opacity: 1 !important;
+}
```

Comment:
- [ ] Same here, Kindly it will be better 😃 if you remove the ```!important``` from the style here as this is from basic CSS bad practices are not used, so The ```!important``` rule was conscripted in the mid-to-late 90’s to help web designers and developers override the rules of their style sheet. The ```!important``` rule💯 prevails over all the previous styles and says, “I’m more important, forget about all of that other CSS and use me!“. Using this out of laziness will give you a headache later on as your style sheet will be harder to maintain.🚀👌You can read [Here](https://speckyboy.com/good-bad-css-practices) about best practices 😊💯